index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
+  several core properties of a <a>DID</a> as an identifier—especially
+  decentralization and cryptographic verifiability—<a>DIDs</a> are generally
+  only meaningful to machines, not humans. To illustrate, compare the following
+  two URIs:
+  </p>
+    <p>
+       <code>https://www.w3.org/2019/did-wg/WorkMode/getting-started</code>
+    </p>
+    <p>
+        <code>did:example:8uQhQMGzWxR8vw5P3UWH1j</code>
+    </p>
+    <p>
+    The first is the URL of the Getting Started page of the W3C DID Working
+    Group. This is a human-meaningful identifier (at least to someone who
+    understands the English language). In this sense, the reader can be said to
+    “know” what the URL identifies without having to dereference it (provided
+    the reader trusts the publisher’s label).
+    </p>
+  <p>
+  The second URI—the example <a>DID</a>—is meaningless to humans no matter what
+  language you speak. What it identifies is anyone’s guess in the absence of
+  further information describing the <a>DID subject</a>. So further information
+  about the <a>DID subject</a> is only discoverable by resolving the <a>DID</a>
+  to the <a>DID document</a>, obtaining a verifiable credential about the
+  <a>DID</a>, or via some other description of the <a>DID</a>.
+  </p>
+    <h2>Does the DID identify the DID document?</h2>
+    <p>
+    No. To be very precise, the <a>DID</a> identifies the <a>DID subject</a> and
+    <em>resolves to</em> the <a>DID document</a> (by following the protocol
+    specified by the <a>DID method</a>). The <a>DID document</a> is not a
+    separate resource from the <a>DID subject</a> and does not have a URI
+    separate from the <a>DID</a>. Rather the <a>DID document</a> is an artifact
+    of <a>DID resolution</a> controlled by the <a>DID controller</a> for the
+    purpose of describing the <a>DID subject</a>.
+    </p>
+    <p>
+    This distinction is illustrated by the graph model shown in figure 2.
+    </p>
+        <figure id="did-and-did-document-graph">
+          <img style="margin: auto; display: block; width: 75%;"
+            src="diagrams/figure-a.1-did-and-did-document-graph.png" alt="
+Diagram showing a graph model for how DID controllers assign DIDs to identify
+DID subjects and resolve to DID documents that describe the DID subjects.
+          " >
+          <figcaption>
+              A <a>DID</a> is an identifier assigned by a <a>DID controllers</a>
+              to identifies a <a>DID subject</a> and resolve to a
+              <a>DID document</a> that describes the <a>DID subject</a>. The
+              <a>DID document</a> is an artifact of <a>DID resolution</a> and
+              not a separate resource distinct from the <a>DID subject</a>.
+          </figcaption>
+        </figure>
+
+    <h2>How does the DID document describe the DID subject?</h2>
+    <p>
+    Each property in a <a>DID document</a> is a statement by the
+    <a>DID controller</a> that describes:
```

Comment:
```suggestion
    <h2>What does the DID document say about the DID subject?</h2>
    <p>
    Each property in a <a>DID document</a> is a statement by the
    <a>DID controller</a> that refers to:
```

Having a bit of trouble with "describes" language, which falls a bit too close to PII territory